react/components/BuyButton/index.js

```diff
@@ -102,17 +105,23 @@ export class BuyButton extends Component {
                 {this.props.children}
               </Button>
             ) : (
-              <Button primary onClick={() => this.handleAddToCart(client)}>
-                {this.props.children}
-              </Button>
-            )}
+                <Button primary onClick={() => this.handleAddToCart(client)}>
+                  {this.props.children}
+                </Button>
+              )}
           </div>
         )}
       </ApolloConsumer>
     )
   }
 }
 
+BuyButton.defaultProps = {
+  isOneClickBuy: false,
+  quantity: 1,
+  seller: 1,
+}
+
 BuyButton.propTypes = {
```

Comment:
Move this and the default props to static attribute

Reply:
Why?

Reply:
This way the code is easier to read and looks less messy!

Reply:
what's your metric to 'easier'?

Reply:
I mean that this isn't really a part of the code that we will be changing a lot or affects the functionality, the way I see it is when putting the propTypes inside the class you gonna make a lot bigger and harder to read, therefore removing the focus from what really matters, which is the render function and the others functions that controls state and events.

Reply:
I rather put it in another file then inside of the class it self.

Reply:
yeah, but I only really asked to change because of the other components, for consistency